compiler-rt/lib/scudo/standalone/mem_map_linux.cpp

```diff
@@ -129,7 +129,7 @@ void MemMapLinux::releaseAndZeroPagesToOSImpl(uptr From, uptr Size) {
 bool ReservedMemoryLinux::createImpl(uptr Addr, uptr Size, const char *Name,
                                      uptr Flags) {
   ReservedMemoryLinux::MemMapT MemMap;
-  if (!MemMap.map(Addr, Size, Name, Flags | MAP_NOACCESS))
```

Comment:
I think the choice of whether to set `MAP_NOACCESS` or not can be left to the caller, like we do in `ReservedMemoryFuchsia::createImpl`